lib/compat/wordpress-6.1/block-patterns/footer-with-large-font-size.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Footer: Footer with Large Font Size.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'      => _x( 'Footer with Large Font Size', 'Block pattern title' ),
+	'blockTypes' => array( 'core/template-part/footer' ),
+	'categories' => array( 'footer' ),
+	'content'    => '<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"15px","padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"left"}} -->
```

Comment:
**Not a blocker of course,** but his inconsistency of some patterns using em and this pattern using px for the spacing is a bit strange.

Reply:
I'll defer to @beafialho again. :)

Reply:
I usually work with px. Can we apply the equivalent em values?